product_pack/__manifest__.py

```diff
@@ -2,11 +2,12 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Product Pack",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Product",
     "summary": "This module allows you to set a product as a Pack",
     "website": "https://github.com/OCA/product-pack",
-    "author": "NaN·tic, ADHOC SA, Tecnativa, Odoo Community Association (OCA)",
+    "author": "NaN·tic, ADHOC SA, Tecnativa,bloopark systems GmbH & Co. KG, "
```

Comment:
@Bruno-Zanotti If I understood correctly what you said in the author section, bloopark should be removed from this manifest key